doc/bootstrap.rst

```diff
@@ -162,14 +162,32 @@ Deploying RGWs
 
 Cephadm deploys radosgw as a collection of daemons that manage a
 particular *realm* and *zone*.  (For more information about realms and
-zones, see :ref:`multisite`.)  To deploy a set of radosgw daemons for
-a particular realm and zone,::
-
-  # ceph orch apply rgw *<realm-name>* *<zone-name>* *<num-daemons>* [*<host1>* ...]
+zones, see :ref:`multisite`.)
 
 Note that with cephadm, radosgw daemons are configured via the monitor
 configuration database instead of via a `ceph.conf` or the command line.  If
 that confiruation isn't already in place (usually in the
 ``client.rgw.<realmname>.<zonename>`` section), then the radosgw
 daemons will start up with default settings (e.g., binding to port
 80).
+
+If a realm has not been created yet, then the first step is to create a new realm,::
+
+  # radosgw-admin realm create --rgw-realm=<realm-name> --default
+
+The next step would be to create a new zonegroup,::
```

Comment:
Next, create a new zonegroup::

Reply:
Does it make sense to suggest names for the zonegroup? I think in the absence of any context, it's not obvious what sort of name to choose.  Maybe "myorg" for realm, "default" for zonegroup, and "us-east-1" for zone, or something?

Reply:
Can do these. Should we add an example for all of these commands? The one example at the end doesn't have a zonegroup.